.gitignore

```diff
@@ -0,0 +1,2 @@
+bin/
```

Comment:
you dont want to check in the bin and obj file to github.  You just want to check in your code and leave out the stuff that the compiler generates for you.   The way to do this is create a `.gitignore` file and list all the stuff you want git to ignore :-)